frontend/document/documentregistry.lua

```diff
@@ -21,6 +22,7 @@ function DocumentRegistry:addProvider(extension, mimetype, provider, weight)
         weight = weight or 100,
     })
     self.filetype_provider[extension] = true
+    self.mimetype_provider[mimetype] = extension
```

Comment:
I guess you named it mimetype_provider for symetry with filetype_provider - but mimetype_provider does not return a provided, so mimetype_ext feels what it should be?